packages/core/src/App/Containers/CloseUKAccountModal/close-uk-account-modal.jsx

```diff
@@ -0,0 +1,121 @@
+                    </Text>
+                    <Text size={isMobile() ? 'xxs' : 'xs'} as='p' line_height='s' className='dc-text__bullet_desc'>
+                        <Localize i18n_default_text='You will lose access to your account when it gets closed, so be sure to withdraw all your funds.' />
+                    </Text>
+                </li>
+            </ol>
+        </>
+    );
+
+    return (
+        <>
+            <div className='close-uk-account-header'>
+                <Text
+                    as='h1'
+                    weight='bold'
+                    className='close-uk-account-header__heading'
+                    align='left'
+                    size={isMobile() ? 'xxs' : 's'}
+                >
+                    <Localize i18n_default_text='Your account is scheduled to be closed' />
+                </Text>
+            </div>
+            <div className='close-uk-account__content'>
+                <Text as='p' size={isMobile() ? 'xxs' : 'xs'}>
+                    <Localize i18n_default_text='As part of the changes in our market offerings, we will be closing our UK clients’ accounts.' />
+                </Text>
+                <br />
+                <Text align='left' weight='bold' as='p' size={isMobile() ? 'xxs' : 'xs'}>
+                    <Localize i18n_default_text='What this means for you' />
+                </Text>
+                <Text as='p' size={isMobile() ? 'xxs' : 'xs'}>
+                    <Localize i18n_default_text='From 25 Apr 2022 onwards, you won’t be able to trade on Deriv. Also, you won’t be able to make deposits into your Deriv account. You need to withdraw any funds from your account as soon as possible.' />
+                </Text>
+                <br />
+                <Text align='left' weight='bold' as='p' size={isMobile() ? 'xxs' : 'xs'}>
+                    <Localize i18n_default_text='What you need to do now' />;
+                </Text>
+                <BulletData />
+            </div>
+            <div className='close-uk-account dc-dialog__footer'>
+                <Button
+                    className='continue-button'
+                    onClick={() => {
+                        showCloseUKAccountPopup(false);
+                        removeNotificationMessageByKey({ key: 'close_uk_account' });
+                    }}
+                    primary
+                    large
+                >
+                    <Localize i18n_default_text='OK, I understand' />
+                </Button>
+            </div>
+        </>
+    );
+};
+
+const CloseUKAccountModal = ({
+    is_logged_in,
+    is_loading,
+    is_close_uk_account_modal_visible,
+    removeNotificationMessageByKey,
+    showCloseUKAccountPopup,
+}) => {
+    const [is_visible, setVisible] = React.useState(false);
+
+    React.useEffect(() => {
+        if (is_logged_in && is_close_uk_account_modal_visible) {
```

Comment:
We should simplify this condition to `setVisible(is_logged_in && is_close_uk_account_modal_visible);`